### PR TITLE
Doischer/3166 account recalculate bulk action

### PR DESCRIPTION
**Pull request by tomdoischer**
https://redmine.czechidm.com/issues/3166

### CODE REVIEW

**Review by kucera-roman — COMMENTED** [review]
Provisioning for account is executed multiple times, when user has multiple roles (e.g permission roles)

This is caused by IdentityProvisioningExecutor::doProvisioning 
For use case when User has one login role and one permission role the code will find two results and execute provisioning twice for same account.

Maybe improve the condition in IdentityProvisioningExecutor or use different method in AccountProvisioningBulkAction for performing provisioning.

Otherwise LGTM

**Review by kucera-roman — CHANGES_REQUESTED** [review]
Just changing to request change :)

File: Realization/frontend/czechidm-acc/src/locales/en.json [hunk 1]
```diff
@@ -2577,6 +2577,11 @@
         "label": "Provision without account recalculation",
         "help": "This operation invokes <strong>provisioning</strong> for each selected users. Provisioning will be performed <strong>asynchronously</strong>. You can see progress of this operations in the <strong>Events</strong> agenda."
       },
+      "acc-account-provisioning-bulk-action": {
+        "icon": "link",
+        "label": "Provision and recalculate an account without recalculation",
```
[the remark this answers is not in the document]
Comment (tomdoischer) [reply Realization/frontend/czechidm-acc/src/locales/en.json]: Thanks, I made the changes.

**Review by kucera-roman — COMMENTED** [review]
LGTM, works nicely, thx

**Pull request merged**